marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/environment/DownloadProjectFilesMvcTest.java

```diff
@@ -21,7 +21,8 @@
 
 public class DownloadProjectFilesMvcTest extends AbstractMvcTest {
 
-    private final static String PATH = "/api/environment/downloadProjectFiles";
+    private final static String ARTIFACTS_PATH = "/api/environment/downloadProjectUpdates";
```

Comment:
I'll test this out locally tomorrow morning, but I think we need some more renaming here so that we consistently use "project updates" and "project files" - i.e. "artifacts" is no longer meaningful in this context, because it's not clear what artifacts we're getting.

Reply:
Correction, trying it out now, will try to get feedback soon.